src/theme.js

```diff
@@ -0,0 +1,79 @@
+export const theme = {
+  borderRadius: {
+    xl: 'var(--f-borderRadius--xl)',
+    l: 'var(--f-borderRadius--l)',
+    m: 'var(--f-borderRadius--m)',
+    s: 'var(--f-borderRadius--s)',
+  },
+  breakPoint: {
+    sUp: '@media (min-width: 460px)',
```

Comment:
`sUp`? 🤔

Reply:
a bit of an awkward name 😄 . It's for `s` but up. Any better name comes to mind?

Reply:
Oh okay. Not really haha